common/src/main/java/com/wynntils/models/items/annotators/game/TomeAnnotator.java

```diff
@@ -15,14 +15,17 @@
 
 public final class TomeAnnotator implements GameItemAnnotator {
     private static final Pattern TOME_PATTERN = Pattern.compile(
-            "^§[5abcdef]((?<Variant>[\\w']+)? ?Tome of (?<Type>\\w+))( (?<Subtype>.+)( (?<Tier>[IVX]{1,4}))?)?$");
+            "^§[5abcdef](?<Unid>Unidentified )?(?<TomeName>((?<Variant>[\\w']+)? ?Tome of (?<Type>\\w+))( (?<Subtype>.+)( (?<Tier>[IVX]{1,4}))?)?)$");
```

Comment:
By convention we use all lowercase group names in most of our regexes. This one did not follow the "rules". Please fix it now, as you changed it anyways.